debian/copyright

```diff
@@ -1,16 +1,64 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: pinball-table-hurd
 Source: <https://github.com/rzr/pinball-table-hurd>
+ 
+Files: data/hurd/bump.wav
```

Comment:
looks good but maybe we could use a default "*" wildcard for files that align to project licence

Reply:
Maybe, however there is multiple author of different file:
data/hurd/bump.wav and other are from original tux table be Henrik Enqvist, other like data/hurd/floor.png are created by me from scratch.
Which of these do you think should use the default "*"?